package.json

```diff
@@ -21,6 +26,11 @@
     "jquery": "^3.2.1"
   },
   "devDependencies": {
-    "live-server": "^1.2.0"
+    "autoprefixer": "^7.1.1",
```

Comment:
`autoprefixer` Package that works with `postcss` to add vendor prefixes to CSS automatically.